css-ecommerce-page/styles.css

```diff
@@ -0,0 +1,72 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  text-align: center;
```

Comment:
Don't `text-align: center` like this, it's why you have so many random spacings for your prices.